Secretly/Models/Faker.swift

```diff
@@ -8,11 +8,13 @@
 
 import Foundation
 
-struct Faker: Restable {
+struct Faker {
     let username: String
     let email: String
+}
 
-    func identifier() -> String {
-        return ""
+extension Faker: Restable {
+    var id: String {
+        ""
```

Comment:
oops I missed the return
```suggestion
        return ""
```

Reply:
the return is not needed in blocks of a single line. 😃